kn/kn-create.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: kn-create
```

Comment:
Should we also support `-f` on create – to be able to "update" the hard way a service ?